news/2019-03-21-mwc-7k-2019-concludes.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "osu!mania 7K World Cup 2019 Concludes"
+date: 2019-03-21 02:00:00 +0000
+---
+
+Last weekend, the final keys were pressed, and the osu!mania 7K World Cup 2019 came to its grand conclusion. Read on to find out more!
+
+![](/wiki/shared/news/banners/MWC7K_2019_banner.jpg)
+
+86 players. 24 countries. 1 prize.
+
+The past six weeks brought the greatest osu!mania 7K players centre-stage, in a clash of skill, wit and tenacity. Riding off the highly successful 2v2 format introduced last year, coupled with with stronger teams and a freshly minted mappool to match, this year’s world cup was all set for a one-way course to success.
+
+Without further ado, let's get to the results!
+
+![](/wiki/Tournaments/MWC/2019_7K/img/podium.png)
+
+**Team South Korea takes home the gold, defending their osu!mania 7K Champion title for the 3rd year in a row!**
+
+Led by veteran player [Jakads](https://osu.ppy.sh/users/259972) for the third year in a row, Team South Korea showed indomitable will and consistency throughout the tournament. Despite a minor setback in the Semifinals stage, they marched on to claim the championship title that they so rightfully deserve. A splendid performance - we expected no less.
```

Comment:
```suggestion
Led by veteran player [Jakads](https://osu.ppy.sh/users/259972) for the 3rd year in a row, Team South Korea showed indomitable will and consistency throughout the tournament. Despite a minor setback in the Semifinals stage, they marched on to claim the championship title that they so rightfully deserve. A splendid performance was given—we expected no less.
```
usually it would be "third", but you have "3rd" in the previous sentence and showing the numeral makes it more clear that the number is important. it's not a grammar rule but I feel like that makes sense lol